src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -657,6 +657,9 @@ bool ApplicationManagerImpl::ActivateApplication(ApplicationSharedPtr app) {
   LOG4CXX_AUTO_TRACE(logger_);
   DCHECK_OR_RETURN(app, false);
 
+  auto& help_prompt_manager = app->help_prompt_manager();
+  help_prompt_manager.OnAppActivated(false);
```

Comment:
better to create constant 
const bool is_restore = false;
or add comment  
help_prompt_manager.OnAppActivated(/*is_restore=*/false);

Reply:
Thanks, fixed in https://github.com/smartdevicelink/sdl_core/pull/2186/commits/aec976d2791859e8699f3728cb3e5d2e29405e89